include/drm/drm_fourcc.h

```diff
@@ -113,6 +113,10 @@ extern "C" {
 
 #define DRM_FORMAT_AYUV		fourcc_code('A', 'Y', 'U', 'V') /* [31:0] A:Y:Cb:Cr 8:8:8:8 little endian */
 
+/* 64 bpp RGB */
+#define DRM_FORMAT_XRGB161616  fourcc_code('X', 'R', '4', '8') /* [63:0] x:R:G:B 16:16:16:16 little endian */
```

Comment:
Whats the use case ?

https://lists.freedesktop.org/archives/dri-devel/2017-July/147426.html not sure why this didnt get merge yet.

Reply:
One of the test cases in CTS trying to create a buffer of HAL_PIXEL_FORMAT_RGBA_FP16 type. For this case it is added.
Please refer https://jira01.devtools.intel.com/browse/OAM-52530 for more details.

I'm not sure why https://lists.freedesktop.org/archives/dri-devel/2017-July/147426.html  didn't get merged.

Reply:
no changes in minigbm or anywhere else ?

Reply:
Changes are required in minigbm and hwcomposer. We need this to be merged first, later I'll submit patch for rest.

Reply:
Do we ship/support any hardware that would support this format? I'm pretty sure that's why such patch has not landed.

Reply:
64 bit RGBX format is supported from Gen9

Reply:
OK, maybe kernel support is missing or then it's about missing userspace.

Reply:
not sure about kernel need to check but in user space components libdrm, Minigbm and hwc support is missing but mesa supports.

Reply:
Right, I guess Mesa currently supports this as a texture format but not as a window buffer format.

Reply:
In kernel also support is missing. The 64 bit format has some limitation we cant use the plane scalar so its not added as part of the plane supported formats. Anyway the cts test is about hardware buffer off-screen test so with the proper user space support, things will work

Reply:
Yeah in context of this test it should be fine to add. I was just wondering why it's still missing, hopefully will get solved in some point.

Reply:
@kalyankondapally , Will this be merged?. Because, I'll have to submit patch for minigbm and hwcomposer yet.